src/Tabs/Tabs.js

```diff
@@ -233,11 +239,13 @@ class Tabs extends Component {
 
     return (
       <div style={prepareStyles(Object.assign({}, style))} {...other}>
-        <div style={prepareStyles(Object.assign(styles.tabItemContainer, tabItemContainerStyle))}>
-          {tabs}
-        </div>
-        <div style={{width: inkBarContainerWidth}}>
-          {inkBar}
+        <div className={labelsContainerClassName}>
```

Comment:
For example adding sticky behavior to the tabs labels so that on scroll the tabs will stick to the top of the screen.
http://stickyjs.com/